#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_update_manpage.dpatch by Laszlo Boszormenyi <gcs@debian.hu>
## 
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add the missing '-r clipboard' option to manpage.

@DPATCH@
--- del/rdesktop-1.6.0/doc/rdesktop.1	2007-01-19 14:47:35.000000000 +0100
+++ rdesktop-1.6.0/doc/rdesktop.1	2011-03-26 13:55:09.806570001 +0100
@@ -215,6 +215,10 @@
 <Vendor Name> - optional device vendor name. For list of examples run
 rdesktop without parameters.
 .TP
+.BR "-r clipboard:[off|PRIMARYCLIPBOARD|CLIPBOARD]"
+Enable clipboard redirection. 'PRIMARYCLIPBOARD' looks at both PRIMARY and
+CLIPBOARD when sending data to server. 'CLIPBOARD' looks at only CLIPBOARD.
+.TP
 .BR "-0"
 Attach to the console of the server (requires Windows Server 2003
 or newer).
